src/main/java/com/statisticalcalculator/CalcDTO/SaveCalcDTO.java

```diff
@@ -0,0 +1,81 @@
+package com.statisticalcalculator.CalcDTO;
+
+public class SaveCalcDTO {

```

Comment:
Since classes are nouns, avoid using verbs in class names.  Is "save" the only thing this DTO can be used to do?  Probably not... I would hope not.